install/migrations/update_10.0.x_to_10.1.0/knowbaseitem.php

```diff
@@ -37,3 +37,33 @@
  * @var array $ADDTODISPLAYPREF
  */
 $ADDTODISPLAYPREF[KnowbaseItem::class] = [79, 131, 13];
+
+$table = 'glpi_knowbaseitems';
+
+$fkey_to_add = 'entities_id';
+if (!$DB->fieldExists($table, $fkey_to_add)) {
+    $migration->addField(
+        $table,
+        $fkey_to_add,
+        "int {$default_key_sign} NOT NULL DEFAULT '0'",
+        [
+            'value' => 0,
+            'after' => 'id'
+        ]
+    );
+    $migration->addKey($table, $fkey_to_add);
+}
+
+$fkey_to_add = 'is_recursive';
```

Comment:
This is not a fkey, variable name should be changed.
```suggestion
$field_to_add = 'is_recursive';
```